data/coalition/coalition missions.txt

```diff
@@ -3111,3 +3111,91 @@ mission "Coalition: Outskirts Gauger Ready"
 			`	While the Arachi work on the outfit, she explains that they used the probe's algorithms to have the scanners excel at finding basic energy signatures, such as heat or energy levels of other ships. "Made to check for complex machinery like outfits, or recognize specific cargo types, it was not, but still find uses for its current form, I'm sure you will."`
 			`	When everything's ready, you two head back inside your ship, and the elevator brings you back up. You fly back to the main landing area, and she pulls close to you just before you open your hatch. "Now, still... experimental, the outfit is, and provide the Heliarchs with a potentially faulty product, we do not wish. So, naturally, not yet registered it with them, we have. Provide you with more copies, we cannot, so careful not to lose this one, you should be." She thanks you again for helping out, and leaves for her workplace.`
 				decline
+
+# TODO: Make Ann undergo the same registration process as
+# First Last entering Coalition space for the first time
+mission "Sativa: Struggling Author 1"
+	minor
+	source "Zug"
+	destination "Ki Patek Ka"
+	to offer
+		random < 30
+		credits > 140
+		has "Coalition: Contributor: done"
+	on offer
+		conversation
+			`	Upon entering the spaceport, you walk past a bookstore, currently holding a book signing event for an author known as 'Ann Oryza Sativa'. Nobody pays attention to her. It appears as if she is a struggling author.`
+			choice
+				`	(Ignore her.)`
+					decline
+				`	(Buy one of her books.)`
+					goto "buy her book"
+				`	(Sit down and talk to her.)`
+			label "talk to her"
+			`	You sit down on the table, next to Ann. She greets you with a smile. "Hello! Would you like to buy one of my books?", she asks.`
+			choice
+				`	"What are your books about?"`
+					goto "what are her books about"
+				`	"No, I'm not interested. How has life been as an author though?"`
+					goto "life as an author sucks"
+			label "what are her books about"
+			`	I mostly write science fiction books.`
```

Comment:
It might be more interesting for her to talk about her books, more than just the genre.

Reply:
d868cfc8854ef3fcca133bc2295583eab74af80a